compiler/noirc_evaluator/src/ssa/opt/rc.rs

```diff
@@ -314,7 +314,7 @@ mod test {
 
         builder.insert_store(v0, v7);
         let v8 = builder.insert_load(v0, array_type);
-        builder.insert_dec_rc(v8);
+        builder.insert_dec_rc(v8, v1);
```

Comment:
Could you update the test comment? Or even better switch this test to use the SSA parser?